roles/ansible_tower/README.md

```diff
@@ -0,0 +1,58 @@
+ansible\_tower
+===========
+
+Deploys a single Ansible Tower instance with a co-located Postgres database
+server
```

Comment:
Do we want to be more indicative of how it does this? E.g. by downloading some other automation and running that?